cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -328,6 +328,19 @@ void SparsePoseGraph::HandleScanQueue() {
           common::MutexLocker locker(&mutex_);
           constraints_.insert(constraints_.end(), result.begin(), result.end());
         }
+
+        // Update the trajectory connectivity structure with the new
+        // constraints.
+        for (Constraint constraint : result) {
+          int trajectory_id_node = constraint.node_id.trajectory_id;
```

Comment:
Inline since there is a single use? Otherwise, this should be `const`.

Reply:
Done